NXTViSy_arduino_firmware.ino

```diff
@@ -0,0 +1,86 @@
+#include <SPI.h>
+#include <Pixy.h>
+#include <PixyViSy.h>
+#include <Wire.h>
+#include <stdint.h>
+
+//#define DBG
```

Comment:
@vargac There probably should be a comment that would explain what `DBG` is and what it does.